tests/testthat/_snaps/caret.md

```diff
@@ -0,0 +1,36 @@
+# can print tidymodels model
+
+    Code
+      v
+    Output
+      
+      -- cars_rf - <butchered_train> model for deployment 
+      A random forest regression model using 3 features
+
+# create plumber.R for xgboost
+
+    Code
+      cat(readr::read_lines(tmp), sep = "\n")
+    Output
+      # Generated by the vetiver package; edit with care
+      
+      library(pins)
+      library(plumber)
+      library(rapidoc)
+      library(vetiver)
+      
+      # Packages needed to generate model predictions
+      if (FALSE) {
+          library(caret)
+          library(dplyr)
+          library(e1071)
```

Comment:
This package being here surprised me; is that expected?

Reply:
It is used to get the out-of-bag statistics.